crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -521,6 +540,7 @@ enum ActionErrorKind {
     ValidatorUpdate(tendermint::error::Error),
     SudoAddressChange(SudoAddressChangeActionError),
     Mint(MintActionError),
+    Ibc(anyhow::Error),
```

Comment:
Can you use `Box<dyn std::error::Error>` here instead? Fortunately `From<anyhow::Error> for Box<dyn std::error::Error>`. Maybe even `dyn Error + Send + Sync` are necessary.

The fact that this never implements clone may or may not be problematic in the future because we need that for `tryhard` (retry logic). We have to see.